### PR TITLE
Update user docs for 0.12 release

### LINKED ISSUE #66: Fix up basic command line instructions
Proposing to improve the guide for users to use the command line...

1. It shouldn't be a Warning box
1. Clarify Windows instructions more, perhaps with a screenshot or using the `WIN+R` shotcut.. I think this part is very hard to understand, also it assumes drive letter `C:`.
   > In Windows you need to open cmd.exe Command prompt in the folder where Kolibri executable is located: c:/Python34/Scripts.
1. I don't think that `python kolibri start` works on MacOS, but maybe `python -m kolibri start` will? It also lacks an instruction on how to start the command line (I think Spotlight + type "Terminal", not sure)
1. Remove the .PEX instructions entirely.
1. Also remove the last part
   > Make sure not to include the angle brackets “< >” in the commands below.*

image

### PR DESCRIPTION
**Pull request by radinamatic**
- [X]  Update all the new UI and feature changes for the Coach dashboard

- [X]  Replace `Exams` for `Quizzes`

- [X]  Update version number 0.11 to 0.12 in installer filenames

- [X]  Explicitly add iOS Safari 9.3+ to the list of supported browsers. Documents [#4676](https://github.com/learningequality/kolibri/pull/4676).

- [X]  Update Device > Info and Facility > Data pages

- [X]  Update all the screenshots with the new appbar

- [X]  Management commands for reordering channels. Documents [#4799](https://github.com/learningequality/kolibri/pull/4799).

- [X]  Clarify command line instructions. Fixes #66

### CODE REVIEW

File: docs/_static/theme_overrides.css [hunk 1]
```diff
@@ -63,13 +63,38 @@ div.figure {
 	color: #fff;
 }
 
+.clock {
+    padding-bottom: 1px !important;
+    /*background-color: #fff;*/
```
Comment (indirectlylit) [inline docs/_static/theme_overrides.css:68]: it's common practice to delete commented-out code before committing it

File: docs/coach.rst [hunk 2]
```diff
@@ -3,115 +3,85 @@
 Coach your Learners in Kolibri
 ##############################
  
-You can track progress of the **Learners**, create and assign **Lessons** and **Exams** to classes or learner groups from the **Coach** dashboard. The default view of the **Coach** dashboard presents the list of **Classes** with their assigned coaches, and the number of learners enrolled in each class.
+You can track progress of the **Learners**, create and assign **Lessons** and **Quizzes** to classes or learner groups from the **Coach** dashboard. The default view of the **Coach** dashboard presents the list of **Classes** with their assigned coaches, and the number of learners enrolled in each class.
 
-Click on a class to access the progress-tracking features and create lessons and exams.
+Click on a class to access the progress-tracking features and create lessons, quizzes and groups.
```
Comment (indirectlylit) [inline docs/coach.rst:8]: do we generally use serial/oxford commas?

File: docs/coach.rst [hunk 3]
```diff
@@ -3,115 +3,85 @@
...
+	  Class Home presents an overview of learner progress and activity.
 
-Navigate the topic tree of each channel to see the progress of the selected learner for the topics, subtopics and individual resources. As you navigate, you will see topics added to the breadcrumbs menu.
 
-In the image below, you can see that the learner **Abby L.** accessed and completed only one of the subtopics from the topic **Addition and Subtraction to 10**: 
 
-	.. figure:: img/learner-report-topic.png
-	  :alt: Navigate between topics in the list, or use the breadcrumb links to move between levels.
+Quizzes
+-------
```
Comment (indirectlylit) [inline docs/coach.rst:30]: I wonder if it's helpful to add a note that in previous versions, quizzes were called exams?

File: docs/coach.rst [hunk 4]
```diff
@@ -3,115 +3,85 @@
...
 
-	  Learner Reports across topics for the selected content channel.
+You can review the progress of learners on the assigned quizzes in the **Quizzes** |exams| block.
 
+* Progress bar will indicate how many learners have started |started| and completed |completed-small| the quiz.
+* You can click on the progress bar itself to open the complete quiz data in the **Reports** tab.
```
Comment (indirectlylit) [inline docs/coach.rst:35]: the word 'itself' seems perhaps unnecessary (same on L44)

File: docs/coach.rst [hunk 5]
```diff
@@ -3,115 +3,85 @@
...
 
-	.. figure:: img/get-5-correct.png
-	    :alt: If the exercise requires 5 correct check marks, learner must provide 5 correct answers one after another. Marks for each answer are located behind the button Check
 
-	    The correct answers (green check marks) in the image above are not in a row; this exercise will be completed only after the learner gives 5 correct answers one after another.
+Class Activity
+--------------
 
-In the example below, the progress bar for the exercise **Subtract within 5** displays 40%, because the student has given 2 correct answers *within the most recent set of 5 attempts*. For example, if this student had made 20 incorrect attempts before the 4 displayed attempts, those 20 incorrect attempts would not get factored into the progress bar.
+As learners interact with lessons and quizzes, you can track their progress in the **Class activity** block in real time.
```
Comment (indirectlylit) [inline docs/coach.rst:51]: super minor note feel free to ignore:

it might be worth thinking of alternative ways of saying "in real time" because...

* it's not truly real-time - there can be up to a 30-second lag
* the phrase "in real time" might not translate easily?

File: docs/install/_linux.rst [hunk 6]
```diff
@@ -32,7 +32,7 @@ Install from a .deb file
 
 The advantages of downloading a ``.deb`` file is the portability: you can copy the file from device to device and install Kolibri without internet access.
 
-#. Download the latest `.deb installer <https://learningequality.org/download/>`_ for Kolibri **version 0.11**, or have it copied to your local drive.
+#. Download the latest `.deb installer <https://learningequality.org/download/>`_ for Kolibri **version 0.12**, or have it copied to your local drive.
```
Comment (benjaoming) [inline docs/install/_linux.rst:35]: Could possibly leave out the version number here for possible merge conflicts and future maintenance.

Comment (benjaoming) [reply docs/install/_linux.rst:35]: Otherwise, we could make a substitution pattern.

File: docs/learn.rst [hunk 7]
```diff
@@ -277,32 +271,32 @@ Click on a lesson title to open its content, and follow the indications of your
...
 
 #. Each time you fill in the answer field, or select one of the available options, the question will be marked by a colored dot in the column on the left side.
 #. You can review and correct your answers to all questions as many times you need before submitting.
-#. Press the *Back to exam list* if you want to pause the exam and come back later. 
-#. Press the button :guilabel:`SUBMIT EXAM` when you want to submit it for grading. 
+#. Press the *Back to quiz list* if you want to pause the quiz and come back later. 
+#. Press the button :guilabel:`SUBMIT QUIZ` when you want to submit it for grading. 
 
-Exam that you did not yet submit will be marked with the |in-progress| icon, so you can easily recognize it, and click to resume when you are ready.
+Quiz that you did not yet submit will be marked with the |in-progress| icon, so you can easily recognize it, and click to resume when you are ready.
```
Comment (benjaoming) [inline docs/learn.rst:297]:
```suggestion
A Quiz that you did not yet submit will be marked with the |in-progress| icon, so you can easily recognize it, and click to resume when you are ready.
```

File: docs/coach.rst [hunk 8]
```diff
@@ -3,115 +3,85 @@
...
+As learners interact with lessons and quizzes, you can track their progress in the **Class activity** block in real time. 
 
-	.. figure:: img/topic-view-detail.png
-	    :alt: 
+* When learners start or complete lessons, you will see one notification displaying the title of the lesson they started |started| or completed |completed-small|, and another displaying the title of the specific resource in that lesson they interacted with. In case of quizzes, you will see only the quiz title notification.
+* When multiple learners are working on the same lesson or quiz, you will see a notification like *Learner1 and 3 other have started...*
+* When learners input an incorrect answer to a question in a quiz or a lesson, the notification in the **Class activity** block will display that one or more learners need help |need-help| with a specific resource.
```
Comment (indirectlylit) [inline docs/coach.rst:55]: > When learners input an incorrect answer to a question in a quiz or a lesson

two notes:

* 'needs help' notifications only occur for exercises in lessons. They do not appear for quizzes
* maybe rephrase to "When learners input multiple incorrect answers..."

File: docs/manage/_advanced.rst [hunk 9]
```diff
@@ -7,15 +7,16 @@ Advanced Manage Options
 Working with Kolibri from the Command Line
 ------------------------------------------
 
-.. warning::
-  * In Windows you need to open ``cmd.exe`` Command prompt in the folder where Kolibri executable is located: ``c:/Python34/Scripts``.
+Tips for Working from the Command Line
```
Comment (indirectlylit) [inline docs/manage/_advanced.rst:10]: header seems a bit redundant with the one above it

File: docs/manage/_advanced.rst [hunk 10]
```diff
@@ -7,15 +7,16 @@ Advanced Manage Options
...
 
-  * On macOS you may need to prefix the commands with ``python``, for example ``python kolibri start``.
+* In Windows you need to open ``cmd.exe`` Command prompt (for example with the :guilabel:`WIN` + :guilabel:`R` shortcut) in the folder where Kolibri executable is located (most probably ``C:/Python34/Scripts`` when Python is installed on drive ``C:``).
 
-  * If you are running Kolibri with the ``PEX`` file, make sure to substitute the ``kolibri`` in below commands **with the exact name of the file you downloaded** preceded by ``./``. For example, to start Kolibri from the downloaded file ``kolibri-v0.11.pex``, type ``./kolibri-v0.11.pex start``.
+* On macOS open Spotlight and type ``Terminal``. You may also need to prefix the commands with ``python -m``, for example ``python -m kolibri start``.
 
-  * Make sure not to include the angle brackets “< >” in the commands below.*
+* If you are running Kolibri with the ``PEX`` file, make sure to substitute the ``kolibri`` in below commands **with the exact name of the file you downloaded** preceded by ``./``. For example, to start Kolibri from the downloaded file ``kolibri-v0.12.pex``, type ``./kolibri-v0.12.pex start``.
```
Comment (indirectlylit) [inline docs/manage/_advanced.rst:17]: >  with the `PEX` file

maybe 

> with the `.pex` file

File: docs/manage/_advanced.rst [hunk 11]
```diff
@@ -7,15 +7,16 @@ Advanced Manage Options
...
 
+* Make sure not to include the angle brackets “< >” in the commands below.
```
Comment (indirectlylit) [inline docs/manage/_advanced.rst:19]: Maybe

> In the commands below, angle brackets and the text between them (`<...>`) are used to denote placeholders for you to modify. Make sure to replace them with your own information.

File: docs/manage/_groups.rst [hunk 12]
```diff
@@ -2,55 +2,52 @@
...
+""""""""""""""""""
 
 To create a new learner group, follow these steps.
 
 #. Click the :guilabel:`NEW GROUP` button.
 #. Give group a desired name.
 #. Click :guilabel:`SAVE` to confirm, or :guilabel:`CANCEL` to exit without creating a group.
+   
 
+Enroll Learners in Group
```
Comment (indirectlylit) [inline docs/manage/_groups.rst:26]: would add 'a':

> Enroll Learners in **a** Group

File: docs/manage/_groups.rst [hunk 13]
```diff
@@ -2,55 +2,52 @@
...
+""""""""""""""""""""""""
 
-Assign Learners to Group
-------------------------
+#. Click to open the desired group in the **Groups** sub-tab.
+#. Click the :guilabel:`ENROLL LEARNERS` button.
 
-Below the existing groups there is a list with all learners currently **NOT** assigned to any groups.
+    * List in this view contains all the learners currently **NOT** enrolled in the selected group.
```
Comment (indirectlylit) [inline docs/manage/_groups.rst:32]: maybe lower-case 'not'? seems like it's yelling at me :)

File: docs/manage/_quizzes.rst [hunk 14]
```diff
@@ -0,0 +1,117 @@
+
+.. _manage_quizzes:
+
+Manage Quizzes
+--------------
+
+You can view, create and delete quizzes, as well as assign them to learners, using the **Quizzes** sub-tab in the **Plan** tab (|edit| icon) of your **Coach** dashboard. Default view displays the list of all quizzes in a selected class, with indicators of their visibility status and groups they are assigned to.
```
Comment (indirectlylit) [inline docs/manage/_quizzes.rst:7]: > **The** default view displays the list of all quizzes...

File: docs/manage/_advanced.rst [hunk 15]
```diff
@@ -7,15 +7,16 @@ Advanced Manage Options
...
-.. warning::
-  * In Windows you need to open ``cmd.exe`` Command prompt in the folder where Kolibri executable is located: ``c:/Python34/Scripts``.
+Tips for Working from the Command Line
+**************************************
 
-  * On macOS you may need to prefix the commands with ``python``, for example ``python kolibri start``.
+* In Windows you need to open ``cmd.exe`` Command prompt (for example with the :guilabel:`WIN` + :guilabel:`R` shortcut) in the folder where Kolibri executable is located (most probably ``C:/Python34/Scripts`` when Python is installed on drive ``C:``).
 
-  * If you are running Kolibri with the ``PEX`` file, make sure to substitute the ``kolibri`` in below commands **with the exact name of the file you downloaded** preceded by ``./``. For example, to start Kolibri from the downloaded file ``kolibri-v0.11.pex``, type ``./kolibri-v0.11.pex start``.
+* On macOS open Spotlight and type ``Terminal``. You may also need to prefix the commands with ``python -m``, for example ``python -m kolibri start``.
```
Comment (benjaoming) [inline docs/manage/_advanced.rst:15]: Is "open Spotlight" a good instruction?

File: docs/manage/_advanced.rst [hunk 16]
```diff
@@ -7,15 +7,16 @@ Advanced Manage Options
 Working with Kolibri from the Command Line
 ------------------------------------------
 
-.. warning::
-  * In Windows you need to open ``cmd.exe`` Command prompt in the folder where Kolibri executable is located: ``c:/Python34/Scripts``.
+Tips for Working from the Command Line
+**************************************
 
-  * On macOS you may need to prefix the commands with ``python``, for example ``python kolibri start``.
+* In Windows you need to open ``cmd.exe`` Command prompt (for example with the :guilabel:`WIN` + :guilabel:`R` shortcut) in the folder where Kolibri executable is located (most probably ``C:/Python34/Scripts`` when Python is installed on drive ``C:``).
```
Comment (benjaoming) [inline docs/manage/_advanced.rst:13]: I'm pretty sure that `WIN+R` + `type "cmd"` is a universal instruction.

File: docs/manage/_advanced.rst (hunk 10 above)
Comment (benjaoming) [inline docs/manage/_advanced.rst:17]: Adding more hard-coded version strings seems problematic.

File: docs/manage/_quizzes.rst [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+      :alt: 
+
+  As you keep adding and removing the exercises you will see confirmation messages at the bottom.
+
+5. Use the **Search** feature to look for topics and exercises about a specific subject. Write the search term in the field, press the :guilabel:`ENTER` key or click the |search| button to display the results. 
+   
+    .. figure:: img/search-exam-resources.png
+      :alt: There are 3 filters available to refine the search; click to open each and select one of the options.
+
+  You can filter the search results by *type* of resource (exercise or topic), include only those from a specific *channel*, or exclude :ref:`resources for coaches <coach_resource>`. Click :guilabel:`EXIT SEARCH` button to return to main quiz view.
```
Comment (indirectlylit) [inline docs/manage/_quizzes.rst:36]: this item and the one above it are kind of the same category of action: browsing the topic tree and searching in order to add a content

**Review by benjaoming — CHANGES_REQUESTED** [review]
Wow - so many changes :) Would maybe have liked some smaller PRs before the big push leading up to a release. But let's totally fast track all these positive changes! I have one big concern, and that's if this would be a good time to implement proper version string substitution and get rid of all the hard-coded version strings.

File: docs/manage/_quizzes.rst [hunk 18]
```diff
@@ -0,0 +1,117 @@
...
+
+6. When you finish selecting resources, click :guilabel:`CONTINUE` button to preview the quiz. 
+
+    .. warning:: Beware that if you click the :guilabel:`GO BACK` button, you will exit the quiz builder and loose the resource selection.
+
+7. You can choose between **Randomized** and **Fixed** question order.
+   
+   * To present the questions in a different (*random*) order to each learner, select :guilabel:`Randomized`. You can not change the order of questions established by the quiz builder, but you can click the *Choose a different set of questions* link, to obtain an alternative one.
```
Comment (indirectlylit) [inline docs/manage/_quizzes.rst:44]: I'd recommend giving the 'Choose a different set of questions' function its own description, separate from the randomized vs fixed toggle:

* 'choose different questions' is something that helps the coach select questions to display
* fixed vs random affects how learners interact with the selected questions

File: docs/manage/_quizzes.rst [hunk 19]
```diff
@@ -0,0 +1,117 @@
...
+    :alt: 
+
+.. note::
+  To manage **Quizzes** |exams| in Kolibri classes and groups you must sign in as **coach**, **admin** or **super admin**.
+
+
+Create New Quiz
+"""""""""""""""
+
+To create a new quiz |exams|, follow these steps.
```
Comment (indirectlylit) [inline docs/manage/_quizzes.rst:19]: The list below is presented as an ordered sequence of actions, but the new UI is designed such that the user can actually perform them in any order. For example, you might select some content, go to preview, choose how many questions you want, go back to select more content, go forward and set a name, and then change question order

I think it would be preferable to de-emphasize the idea that they need to perform the steps in a specific order

File: docs/manage/_quizzes.rst [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+#. Click :guilabel:`SAVE` to confirm, or :guilabel:`CANCEL` to exit without change.
+
+  .. figure:: img/exam-visibility.png
+    :alt: 
+
+
+Change Quiz Status
+""""""""""""""""""
+
+Once you set the visibility of quiz to the chosen group(s) of learners, you need to **Activate** it so it can appear in the **Learn** view of the learners it is assigned to.
```
Comment (indirectlylit) [inline docs/manage/_quizzes.rst:76]: might want to explicitly say that quizzes are not active by default. This still trips me up sometimes

File: docs/manage/_quizzes.rst [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+#. Click :guilabel:`OPTIONS` for the desired quiz and select **Copy quiz**.
+
+    .. figure:: img/copy-exam.png
+      :alt: Use the radio buttons to select the class where you want to copy the quiz to.
+
+#. Select the class to which you wish to copy the quiz to and click :guilabel:`CONTINUE`.
+#. If the destination class has more than one group, select **Entire class** or the group(s) of learners to whom you wish to assign the lesson.
+#. Click :guilabel:`COPY` to confirm, or :guilabel:`CANCEL` to exit without change.
+
+If the quiz is copied to another group in the same class, it will appear in the **Quizzes** tab as the *Copy of...*. Follow the steps in the next heading to rename it.
```
Comment (indirectlylit) [inline docs/manage/_quizzes.rst:97]: > in the next heading

to 

> in the next section

**Review by indirectlylit — APPROVED** [review]
looks great!

no blocking requests here, but a bunch of small notes

**Comment by indirectlylit** [comment]
> I have one big concern, and that's if this would be a good time to implement proper version string substitution and get rid of all the hard-coded version strings.

@benjaoming I agree with this idea, but maybe it would be better to open a new issue for it?

(seems out of scope in the context of this PR)

**Comment by radinamatic** [comment]
@benjaoming Please, do file an issue to change the hardcoded strings, and I'll address them next week after the conference, together with all the notes by @indirectlylit. 

If there's no major gaffe or stuff missing in this PR, let's merge it so it's ready for the blog announcement tomorrow.

**Comment by benjaoming** [comment]
Yes, I can see you 2 pair-programming that feature :) thanks for responding!

**Review by benjaoming — APPROVED** [review]
